prov/usnic/src/usdf_fabric.c

```diff
@@ -309,7 +316,7 @@ usdf_fill_info_dgram(
 
 		fi->handle = hints->handle;
 	} else {
-		fi->mode = USDF_DGRAM_SUPP_MODE;
+		fi->mode = USDF_DGRAM_REQ_MODE;
```

Comment:
I thought we decided that we could not return both PREFIX and PREFIX2...?  (or am I reading this code wrong?)